devel/plugins/event-metadata-plugin.rst

```diff
@@ -0,0 +1,101 @@
+.. _event-metadata-plugin:
+
+=====================
+Event Metadata Plugin
+=====================
+
+.. versionadded:: 8.0
+
+
+Zeek's plugin API allows adding metadata to Zeek events which are then
+transported across a Zeek cluster for consumption by other Zeek nodes.
```

Comment:
Doesn't it also allow adding metadata for anything locally published, regardless of where the event ends up?